pkg/operator/ceph/csi/operator_driver.go

```diff
@@ -78,7 +78,7 @@ func (r *ReconcileCSI) createOrUpdateDriverResources(cluster cephv1.CephCluster,
 
 func (r *ReconcileCSI) createOrUpdateRBDDriverResource(cluster cephv1.CephCluster, clusterInfo *cephclient.ClusterInfo) error {
 	resourceName := fmt.Sprintf("%s.rbd.csi.ceph.com", r.opConfig.OperatorNamespace)
-	spec, err := r.generateDriverSpec(cluster)
+	spec, err := r.generateDriverSpec(cluster, "rbd")
```

Comment:
why not added it below after generating? instead of passing known string and adding it?